src/Sarif.Multitool/MergeCommand.cs

```diff
@@ -11,9 +11,16 @@
 
 namespace Microsoft.CodeAnalysis.Sarif.Multitool
 {
-    internal static class MergeCommand
+    internal class MergeCommand
     {
-        public static int Run(MergeOptions mergeOptions)
+        private IFileSystem _fileSystem;
```

Comment:
>private [](start = 8, length = 7)

`readonly`, here and in all the command classes. (See? I only said it once this time.)

Reply:
i make this change only because you care. i honestly don't see how applying 'readonly' assists in more maintainable code. it just comes across as busy work. can you explain the substantive issue we are protecting against? nice job only making a single comment, btw. :)

---
In reply to: [229508070](https://github.com/Microsoft/sarif-sdk/pull/1125#discussion_r229508070) [](ancestors = 229508070)

Reply:
In reply to a comment of yours that came across in email, but which I think you've since edited out: I make this a habit to document to the maintainer that the value should not be changed, and to prevent the author from accidentally changing it. It costs me nothing to add the keyword.

Reply:
I did edit that out, because on reflection i also see the value. this case is similar to another standard i like to enforce, that methods that fail to access the 'this' pointer are declared static. it is helpful to be explicit and precise even in these small things. i find 'readonly' a thorn because I don't have an internal habit to think about it. i certainly can build that habit, though. resolved!

---
In reply to: [229518035](https://github.com/Microsoft/sarif-sdk/pull/1125#discussion_r229518035) [](ancestors = 229518035)